### PR TITLE
i18n(id): complete missing Indonesian translations

### LINKED ISSUE #1450: Indonesian admin catalog has untranslated strings in current repo state
## Summary

The Indonesian (id) admin catalog is enabled and wired correctly, but the current repository state still has untranslated strings in packages/admin/src/locales/id/messages.po.

## Current gaps

The following entries are currently empty in the Indonesian catalog:

- All authors
- Archived
- Date field to filter on
- Editor / Reporter / Photographer (multiline placeholder)
- Filter by author
- From date
- Insert HTML
- to
- To date

## Notes

This appears to be translation drift rather than an i18n infrastructure issue:

- id is already enabled in packages/admin/src/locales/locales.ts
- locale loading and config look correct
- changelog entries mention prior Indonesian completion, but newer UI strings were added later

## Proposed fix

- Fill the missing Indonesian translations using standard Indonesian UI terminology
- Verify the catalog loads correctly
- Add a small guard test so empty msgstr entries in id/messages.po are easier to catch in future

### PR DESCRIPTION
**Pull request by ahliweb**
## What does this PR do?

Translates the previously identified Indonesian admin i18n gaps using standard Indonesian UI wording, rechecks the catalog, and completes the remaining missing `id` translations currently present on `upstream/main`.

This PR also adds a regression test to catch future empty `msgstr` entries in the Indonesian catalog.

Closes #1450

## Type of change

- [ ] Bug fix
- [ ] Feature (requires [maintainer-approved Discussion](https://github.com/emdash-cms/emdash/discussions/categories/ideas))
- [ ] Refactor (no behavior change)
- [x] Translation
- [ ] Documentation
- [ ] Performance improvement
- [ ] Tests
- [ ] Chore (dependencies, CI, tooling)

## Checklist

- [x] I have read [CONTRIBUTING.md](https://github.com/emdash-cms/emdash/blob/main/CONTRIBUTING.md)
- [x] `pnpm typecheck` passes
- [x] `pnpm lint` passes
- [x] `pnpm test` passes (or targeted tests for my change)
- [x] `pnpm format` has been run
- [x] I have added/updated tests for my changes (if applicable)
- [x] User-visible strings in the admin UI are [wrapped for translation](https://github.com/emdash-cms/emdash/blob/main/CONTRIBUTING.md#internationalization-i18n) (if applicable). Do not include `messages.po` changes except in translation PRs — a workflow extracts catalogs on merge to `main`.
- [x] I have added a [changeset](https://github.com/emdash-cms/emdash/blob/main/CONTRIBUTING.md#changesets) (if this PR changes a published package)
- [ ] New features link to an approved Discussion: https://github.com/emdash-cms/emdash/discussions/...

## AI-generated code disclosure

- [x] This PR includes AI-generated code — model/tool: OpenCode (GPT-5.4)

## Screenshots / test output

- `pnpm typecheck`
- `pnpm lint`
- `pnpm test`
- `pnpm format`

## Notes

- The workflow started from an i18n audit, created issue #1450 to track the work, then completed the translation fix on a branch rebuilt directly from the current `upstream/main` so the final diff stayed atomic.
- The Indonesian catalog now has no empty translatable `msgstr` entries for this codebase state, and the added test guards that invariant.

### CODE REVIEW

**Comment by ahliweb** [comment]
Maintainers: review requested when convenient. This replacement PR is rebuilt directly from the current upstream/main so the diff stays atomic and translation-only.

File: packages/admin/tests/lib/locales.test.ts [hunk 1]
```diff
@@ -22,6 +23,22 @@ test("loadMessages falls back to English for unknown locale", async () => {
...
+		if (msgstrIndex === -1) return false;
+		const nextLine = lines[msgstrIndex + 1];
+		return (
+			lines[msgstrIndex] === 'msgstr ""' && (nextLine === undefined || !nextLine.startsWith('"'))
+		);
+	});
+
+	expect(untranslated).toEqual([]);
+});
+
```
Comment (ascorbic) [inline packages/admin/tests/lib/locales.test.ts:41]: Can you not add tests for specific languages like this. This will mean any PR will fail if it adds strings.

**Review by ascorbic — CHANGES_REQUESTED** [review]

File: packages/admin/tests/lib/locales.test.ts (hunk 1 above)
Comment (ahliweb) [reply packages/admin/tests/lib/locales.test.ts:41]: You're right - I should not add a test tied to a specific language catalog like this, since it would cause unrelated PRs that add strings to fail. Sorry, I didn't think through the impact on other locales. I'll update the PR accordingly, and going forward I'll keep Indonesian translation updates scoped to the Indonesian .po file only.

**Comment by ahliweb** [comment]
This PR has been cancelled and its branch/work has been removed. The related issue #1450 is also being closed because this work has been superseded by the new replacement PR #1496.
